src/olympia/users/tests/test_admin.py

```diff
@@ -47,20 +47,68 @@ def setUp(self):
             'admin:users_userprofile_delete', args=(self.user.pk,)
         )
 
-    def test_search_for_multiple_users(self):
+    def test_search_by_email_simple(self):
         user = user_factory(email='someone@mozilla.com')
         self.grant_permission(user, 'Users:Edit')
         self.client.force_login(user)
         another_user = user_factory()
         response = self.client.get(
             self.list_url,
-            {'q': f'{self.user.pk},{another_user.pk},foobaa'},
```

Comment:
Searching by ids _and_ email at the same time is no longer supported for performance reasons.